internal/ethapi/backend.go

```diff
@@ -89,7 +89,12 @@ type Backend interface {
 
 	// eth/filters needs to be initialized from this backend type, so methods needed by
 	// it must also be included here.
-	filters.Backend
```

Comment:
I changed this to avoid circular import. This happened because I had to import `internal/ethapi` for the `RPCTransaction` type.